### PR TITLE
Task 75105: JVM 17, gradle 8.5, migrate to buildSrc, import font, config dimensity, theme, splash screen

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- [#Task 75105: JVM 17, gradle 8.5, migrate to buildSrc, import font, config dimensity, theme, splash screen](https://edu-redmine.sun-asterisk.vn/issues/75105)
## WHAT
- 
## Evidence (Screenshot or Video)

Snipaste_2024-04-19_10-33-38

## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://sal.vn/WtYPAy |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://sal.vn/piPoQn |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -4,15 +4,46 @@
...
+        android:text="@string/author"
+        android:textColor="@color/white"
+        android:textStyle="italic"
+        android:fontFamily="@font/alexbrush_regular"
+        android:textSize="@dimen/sp_24"
+        android:layout_marginBottom="@dimen/dp_66"
+        app:layout_constraintBottom_toBottomOf="parent"
+        android:gravity="center"
+        />
 </androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (ghost) [inline app/src/main/res/layout/activity_main.xml]: .

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@drawable/ic_launcher_background" />
-    <foreground android:drawable="@drawable/ic_launcher_foreground" />
-    <monochrome android:drawable="@drawable/ic_launcher_foreground" />
+    <background android:drawable="@color/ic_launcher_background"/>
+    <foreground android:drawable="@drawable/ic_launcher_adaptive_fore"/>
 </adaptive-icon>
```
Comment (ghost) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml]: .

File: app/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -2,4 +2,15 @@
...
+    <color name="action_color">#FF7029</color>
+    <color name="light_primary">#CAEAFF</color>
+    <color name="dark_frame">#16181F</color>
+    <color name="dark_shape">#20232D</color>
+    <color name="text_dark_color">#1B1E28</color>
+    <color name="gray_subtext_color">#7D848D</color>
+    <color name="line_stroke_color">#F8F9FA</color>
+    <color name="selection_card_bg">#FFFFFF</color>
+    <color name="onclick_selection_card_bg">#BFBCBC</color>
 </resources>
```
Comment (ghost) [inline app/src/main/res/values/colors.xml]: .

File: app/src/main/res/values/ic_launcher_background.xml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="ic_launcher_background">#24BAEC</color>
+</resources>
```
Comment (ghost) [inline app/src/main/res/values/ic_launcher_background.xml]: .

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -1,3 +1,12 @@
...
+    <string name="onboarding_1_title">Life is short and the\nworld is wide</string>
+
+    <string name="onboarding_1_description">At  Friends tours and travel, we customize\nreliable and trutworthy educational tours to\ndestinations all over the world</string>
+    <string name="onboarding_2_description">To get the best of your adventure you just\nneed to leave and go where you like, we are\nwaiting for you</string>
+    <string name="skip_btn">Skip</string>
+    <string name="get_started_btn">Get Started</string>
+    <string name="let_us_know_you">Let us know you</string>
+    <string name="next_btn">Next</string>
+    <string name="author">\@Hoàng Giang</string>
 </resources>
```
Comment (ghost) [inline app/src/main/res/values/strings.xml]: .

**Review by ghost — APPROVED** [review]

**Review by daolq-2712 — APPROVED** [review]
LGTM!

**Pull request merged**